### ISSUE TITLE
Scientific mode conversion function in toolbox

### ISSUE THREAD

**Issue by madhatter0** [numworks/epsilon#1847]
#### Problem you'd like to fix
I've been doing a fair amount of "physics help" for my nephew recently.  Suppose we're calculating the pressure exerted by a force of 85N on an area of 4.8Mm^2.  He enters 85 / 4.8e6, then presses EXE, and gets 0.00001770833.  When he writes this down in scientific form, quite often the rounding causes errors in the counting of the number of decimal places, and the exponent will be off by one (eg, 1.8x10^-4 Pa, instead of 1.8x10^-5 Pa).

#### Describe the solution you'd like
It would be really helpful to have a toolbox function to express a single numeric answer in scientific mode (and possibly, also, engineering mode).

#### Describe alternatives you've considered
We could put the calculator into scientific mode, but it takes 13 keystrokes to put it in, and another 13 to put it back after, so in practice he doesn't do this.  I'm pretty sure a toolbox function can be accessed in much fewer than 26 keystrokes; in any case, it's done inside calculation mode, so there's less of a context shift.

**Comment by RedGl0w** [comment]
> It would be really helpful to have a toolbox function to express a single numeric answer in scientific mode (and possibly, also, engineering mode).

I don't think a function would be the best solution. In the additional results, having a "result format" display would be great (because the different number base are there, and so it's the place for that feature, and also, because it's pretty quick to access to it).
This place would need to display different additional results for one calculation, and so this is another exemple of usage for https://github.com/numworks/epsilon/issues/1818

**Comment by madhatter0** [comment]
I agree.  What do you think is the best way to go forward?  Close this request, and add it to 1818?

**Comment by RedGl0w** [comment]
> I agree. What do you think is the best way to go forward? Close this request, and add it to 1818?

Nope. #1818 is more a "meta" issue for a feature. Yours is a usage of it. The best thing is to let both of them opened :p

**Comment by Ecco** [comment]
That's a good idea! We offer three result formats (Decimal, Scientific and Engineering). Whenever we use either one of these as the 'main' output, we should offer the other two in the additional result window!

**Comment by madhatter0** [comment]
> That's a good idea! We offer three result formats (Decimal, Scientific and Engineering). Whenever we use either one of these as the 'main' output, we should offer the other two in the additional result window!

That would be just perfect, if it's not too much trouble.

**Comment by RedGl0w** [comment]
> That would be just perfect, if it's not too much trouble.

The main question so about this feature is how to display multiple additional results
But we should discuss about that in #1818

**Comment by fmOOmf** [comment]
Amélioration proposée sur l'amelioration:
Que les flèches droite/gauche permettent de decrémenter/incrementer l'exposant du resultat complémentaire.
https://github.com/UpsilonNumworks/Upsilon/issues/149
